core/controllers/domain_objects_validator_test.py

```diff
@@ -245,3 +247,63 @@ def test_valid_images_do_not_raises_exception(self) -> None:
             ) as f:
                 files[filename] = f.read()
         domain_objects_validator.validate_suggestion_images(files)
+
+
+class ValidateExplorationChangeTests(test_utils.GenericTestBase):
+    """Tests to validate exploration change coming from frontend"""
+
+    def test_incorrect_exp_state_domain_content(self) -> None:
+        """Tests the state content of exploration change."""
+        incorrect_change_dict = {
+            'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+            'property_name': exp_domain.STATE_PROPERTY_CONTENT,
+            'state_name': 'New state',
+            'old_value': state_domain.SubtitledHtml(
+                'content', '').to_dict(),
+            'new_value': state_domain.SubtitledHtml(
+                'content',
+                '<oppia-noninteractive-image filepath-with-value='
+                '"&quot;abc.png&quot;" caption-with-value="&quot;'
+                '&quot;" alt-with-value="&quot;&quot;">'
+                '</oppia-noninteractive-image>').to_dict()
+        }
+        with self.assertRaisesRegex(
+            Exception, 'Image tag \'alt-with-value\' attribute '
```

Comment:
This is wrong, it suggests that the attribute value is a number. It should instead be: "The length of the ... attribute value should be at least 5 characters."

Reply:
Done